app/services/providers/copy_to_recruitment_cycle_service.rb

```diff
@@ -44,6 +47,9 @@ def copy_courses_to_new_provider(provider, new_provider)
       provider.courses.each do |course|
         new_course = @copy_course_to_provider_service.execute(course: course, new_provider: new_provider)
         courses_count += 1 if new_course.present?
+      rescue
```

Comment:
what are you rescuing?

> Avoid rescuing without specifying an error class.

Reply:
Ya, good point, I was just being sloppy. ⏩ ⏩

Reply:
Just to be clear, anything, it's really just a bail-out clause.

Reply:
it got nothing to do with me being picky... its failing linter

https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/results?buildId=68147&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=59a85588-b0ba-5043-24c4-d9e29d89c6f6&l=65